platform/platform-paxweb-jettyconfig/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -73,4 +74,6 @@
 
     <service interface="org.osgi.framework.hooks.service.EventListenerHook" ref="filterInjector"/>
 
+    <reference-list id="filterList" availability="optional" interface="javax.servlet.Filter" />
```

Comment:
Getting references for `javax.servlet.Filter` seems fine to me because it makes sense to want to add all of these but would want to confirm nothing weird happens with pax web trying to grab them as well.

Reply:
No, shouldn't matter. Services can be picked up by any number of bundles and Pax Web will fail to inject them under anything other than the bundle registering them, which has no web context anyways, so should be a no op.